setup.py

```diff
@@ -191,6 +191,8 @@ def run(self):
                 to_preload_cann = []
 
                 cuda_dependencies = [
+                    "libcublas.so.12",
+                    "libcublasLt.so.12",
                     "libcublas.so.11",
                     "libcublasLt.so.11",
```

Comment:
Also you need to modify 
```
                    "libcudart.so.11.0",
                    "libcurand.so.10",
                    "libcufft.so.10",
``

Reply:
Have you checked the version of libcurand and libcufft? Build the docker image locally , then inspect the image.